test/harmonymodulestest.js

```diff
@@ -34,66 +34,203 @@ var modulesTestFixture = {
     'ES6: Modules': {
         // default exports
         'export default 42;': {
-            type: 'ExportDeclaration',
-            'default': true,
-            declaration: {
-                type: 'Literal',
-                value: 42,
-                raw: '42',
-                range: [15, 17],
+            type: 'Program',
```

Comment:
Any particular reason you're starting at the root in these tests? It seems the test runner will allow it, but the rest of the tests in this fixture only include the first statement in the program body...so unless there's a good reason it'd be nice to retain consistency.